packages/app-store/make/api/subscriptions/listBookings.ts

```diff
@@ -0,0 +1,93 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getHumanReadableLocationValue } from "@calcom/core/location";
+import { getCalEventResponses } from "@calcom/features/bookings/lib/getCalEventResponses";
+import findValidApiKey from "@calcom/features/ee/api-keys/lib/findValidApiKey";
+import { defaultHandler, defaultResponder, getTranslation } from "@calcom/lib/server";
+import prisma from "@calcom/prisma";
+
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const apiKey = req.query.apiKey as string;
+
+  if (!apiKey) {
+    return res.status(401).json({ message: "No API key provided" });
+  }
+
+  const validKey = await findValidApiKey(apiKey, "make");
+
+  if (!validKey) {
+    return res.status(401).json({ message: "API key not valid" });
+  }
+
+  try {
+    const bookings = await prisma.booking.findMany({
```

Comment:
We should make a reusable logic for that and import it for Zapier and here. Otherwise, if we make changes in the fields returned for the bookings we would always need to make those changes in both places.

Reply:
Okay, so move the module to a different location and accept the app name/slug as a function param?

Which dir would you move the module to?

Reply:
also, this can be done in this PR itself or should I open another one?

Reply:
I think we can put the reusable code inside the `_utils` folder in `packages/app-store`. And we should have that booking query from `/listBookings` already covered in this PR, so we can assure it will always return the same for Zapier and Make. 

If the nodeScheduler is just a copy of the zapier one, then let's also make sure to make it reusable and not just copy it

Reply:
thanks! I'll make changes as per this.